.github/ISSUE_TEMPLATE/rest-api-mobile.yml

```
@@ -0,0 +1,40 @@
+name: ✨ REST API MOBILE
```

```suggestion
name: 📱 REST API issue for mobile apps
```

Let's change the emoji to differentiate it a bit from the regular Enhancement request. Also making the title a bit more explicit. What do you think?

---

Totally missed this one, great catch, thanks!